time stamp		# time since system start (microseconds)


# safe_path(n) explained::: "n: 1,3 x_value; n: 2,4 y_value"
float64[] safe_path
float64[] target_position   # x_n, y_n, z_n ###When not the same as requested Masternode will discard this message
